lib/BloqadeNoisy/src/noise_models.jl

```diff
@@ -0,0 +1,110 @@
+"""
+Aquila noise model parameters in JSON format
+"""
+AQUILA = """{
+    "coherent":{
+        "local":{
+            "site x":0.05,"site y":0.05,"detuning":0.5,"Rabi frequency":0.018
+        },
+        "global":{
+            "detuning":0.18,"Rabi frequency":0.008
+        }
+    },
+    "incoherent":{
+        "dephasing rate":0.02,"relaxation rate":0.01
+    },
+    "readout":{
+        "single atom":{"p01":0.01,"p10":0.08}
+    }
+}"""
```

Comment:
Can we use `Configurations.jl` to generate Julia Objects instead of using JSON format explicitly?

You can look at BloqadeSchema as a reference on how to set up these objects.